<launch>

<node pkg="feather" name="status_pub" type="status_sender.py" output="screen">
</node>

<node pkg="feather" name="love_pub" type="love_sender.py" output="screen">
</node>

</launch>
